src/components/styles/YouTube.scss

```
@@ -0,0 +1,94 @@
+@import 'variables';
+
+#intro-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: auto;
+}
+
+#intro-title {
+  font: $font-title;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+#intro-subtitle {
+  font: $font-title;
+  font-size: 26px;
+  margin: 0;
```

some margin might look better?
```suggestion
  margin: 4px;
```